exercises/d2e3.js

```diff
@@ -17,15 +17,20 @@ const alarms = [
 ]
 
 app.get('/alarms', (req, res) => {
-  /**
-   * Responds with all the alarms as a JSON response
-   */
+
+res.json(alarms)
 })
 
 app.get('/alarms/:index', (req, res) => {
-  /**
-   * Responds with the alarm at the given index
-   */
+ 
+  const index =parseInt(req.params.index)
+  
+  if (index >= 0 && index < alarms.length) {
```

Comment:
Nit: Excellent use of [defensive programming](https://www.bbc.co.uk/bitesize/guides/z4cg4qt/revision/1)! You can use some [more syntactic sugar](https://stackoverflow.com/a/3067082) here to make this easier to read.
```suggestion
  if (index in alarms) {
```